sycl/test-e2e/ClusterLaunch/cluster_launch_enqueue_functions.cpp

```diff
@@ -0,0 +1,59 @@
+// REQUIRES: cuda
+// RUN: %{build} -Xsycl-target-backend --cuda-gpu-arch=sm_90 -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/ext/oneapi/experimental/cluster_group_prop.hpp>
+#include <sycl/ext/oneapi/experimental/enqueue_functions.hpp>
+#include <sycl/sycl.hpp>
+
+#include <string>
+
+int main() {
+  using namespace sycl::ext::oneapi::experimental;
+
+  sycl::queue queue;
+  auto computeCapability = std::stof(
+      queue.get_device().get_info<sycl::info::device::backend_version>());
+
+  if (computeCapability >= 9.0) {
```

Comment:
I'd be inclined to change this to
```suggestion
  if (computeCapability < 9.0) {
    printf("Cluster group not supported on this arch, exiting...\n");
    return 0;
  }
```
Instead of making the whole test indented